pronto-rubocop.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
   s.summary = 'Pronto runner for Rubocop, ruby code analyzer'
 
   s.licenses = ['MIT']
-  s.required_ruby_version = '>= 2.3.0'
+  s.required_ruby_version = '>= 2.4.0'
```

Comment:
Can you check if the CI works without this? I'd prefer not to drop the Ruby 2.3 support in a minor patch release (`0.11.1`) as we'll be checking a specific RuboCop version anyway.

Reply:
Hmm, looks like rubocop-hq/rubocop#9515 will be in an upcoming release soon 🤞

I'd rather wait till that PR gets merged, and then pin to a specific commit in the CI configuration and then update it after an official release is done.

Reply:
Never mind, I found a workaround and merged it: see ae2faf83078445e888222a72edf620343146c1d6.

Reply:
Thanks. I noticed it when I got conflict.